msteams-platform/apps-in-teams-meetings/teams-together-mode.md

```diff
@@ -137,17 +107,15 @@ The seat dimensions become the canvas for rendering the participant video stream
     >[!NOTE]
     > To delete a scene you created, select **Delete scene** on the top bar.
 
-1. In the **View in Teams** dialog box, select **Preview in Teams**.
+1. In the **View in Teams** dialog box, select **Preview in Teams**. Selecting **Preview** automatically creates a Microsoft Teams app that can be viewed in the **Apps** page in the Teams Developer Portal. Selecting **Preview** automatically creates an app package that is appmanifest.json behind the scene. As stated earlier, this is abstracted, but you can access the automatically created app package by navigating to **Apps** from the menu.
```

Comment:
Suggestion: 1.Can list down the action after preview selection? 
2. Avoid "navigating" - Use "Go to" as per MSTP.